GenderPayGap.WebUI/Views/AccountCreation/ResentVerificationCode.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@using GenderPayGap.Core
+@model GenderPayGap.WebUI.Models.AccountCreation.ConfirmEmailAddressViewModel
+@{
+    ViewBag.Title = "Confirm your email address - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@if (FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.ReportingStepByStep))
+{
+    @section BeforeMain {
+        @{ await Html.RenderPartialAsync("/Views/ReportingStepByStep/ReportingPageHeader.cshtml"); }
+    }   
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Confirm your email address</h1>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
We could probably remove these four lines of code - I don't think there's any value in having two "row"s

Reply:
Done.

Reply:
Actually, I realise why I put in the extra rows. It was so that the step by step sat better, as it would then start in line with the content rather than alongside the title at the top (similar to how it now looks in the guidance). But this can be reviewed later